cmd/descheduler/app/server.go

```diff
@@ -27,28 +27,30 @@ import (
 	"github.com/spf13/cobra"
 
 	aflag "k8s.io/component-base/cli/flag"
-	"k8s.io/component-base/logs"
 	"k8s.io/klog/v2"
 )
 
 // NewDeschedulerCommand creates a *cobra.Command object with default parameters
 func NewDeschedulerCommand(out io.Writer) *cobra.Command {
-	s := options.NewDeschedulerServer()
+	s, _ := options.NewDeschedulerServer()
```

Comment:
I think it would be a good idea to check a non-nil error here instead of just throwing away the error with an `_`.

Reply:
fixed